devops/openshift/queue-consumers-deploy.yml

```diff
@@ -289,6 +289,7 @@ parameters:
   - name: BUILD_TAG
     value: "0"
   - name: QUEUE_CONSUMERS_SECRET_NAME
+    value: sims-api-creds
```

Comment:
Same from https://github.com/bcgov/SIMS/pull/2152/files#r1282487715

Reply:
Just following the steps follwed in api-deploy.yml, also the name was not self understanding what it is, the value just gives an idea what this secret name is.

Reply:
IMO, the standard is old and does not represent well the change that we did when we introduced the sims-ENV-creds.

Reply:
changed